src/game.rs

```diff
@@ -45,7 +45,29 @@ pub mod flags {
 ///
 /// [http://docs.screeps.com/api/#Game.rooms]: http://docs.screeps.com/api/#Game.rooms
 pub mod rooms {
-    game_map_access!(objects::Room, Game.rooms);
+    use std::collections::HashMap;
+
+    use crate::{local::RoomName, macros::*, objects::Room};
+
+    /// Retrieve the full `HashMap<RoomName, Room>`.
+    pub fn hashmap() -> HashMap<String, Room> {
```

Comment:
Isn't it a `HashMap<RoomName, Room>`?

Reply:
It is - I think I just completely forgot to update the `hashmap` function.

Reply:
Turns out there are some problems with getting a `HashMap<RoomName, Room>` directly from JS - stdweb only has conversions for `HashMap<String, V>`.

I've implemented it relatively inefficiently for now, and opened https://github.com/koute/stdweb/issues/359.